site/compiler/people.test.js

```diff
@@ -365,9 +365,9 @@ describe('site/compiler', () => {
         createStateNavigator(),
       );
 
-      expect(routes.length).to.equal(14);
-      expect(routes[12].filePath).to.equal('about-us/people/alex/index.html');
-      expect(routes[13].filePath).to.equal('about-us/people/sari/index.html');
+      expect(routes.length).to.equal(15);
+      expect(routes[13].filePath).to.equal('about-us/people/alex/index.html');
+      expect(routes[14].filePath).to.equal('about-us/people/sari/index.html');
```

Comment:
Gonna try pushing MVP out first, then hopefully if I get time, i'll rework some of these horrible tests!